eng/targets/Cpp.Common.props

```diff
@@ -15,6 +15,16 @@
     <TestProjectSkipReason>You cannot test native ARM64 projects on a x86/x64 machine</TestProjectSkipReason>
   </PropertyGroup>
 
+  <PropertyGroup Label="Configuration">
+    <Configuration Condition="'$(Configuration)'==''">Debug</Configuration>
+    <VCProjectVersion>15.0</VCProjectVersion>
+    <Keyword>Win32Proj</Keyword>
+    <Platform Condition="'$(Platform)' == ''">x64</Platform>
+    <PlatformToolsetVersion>v143</PlatformToolsetVersion>
```

Comment:
FYI @HaoK
I moved the toolset and windows versions to our common Cpp props and removed them from IIS and Installers. So in case you notice them missing from IIS in the future this is why 😉